### ISSUE TITLE
Feature: do not show balance or history

### ISSUE THREAD

**Issue by gotcha** [breez/breezmobile#145]
For privacy reasons, I think the UI should not start with showing balance and history.

Reasoning, the main use case for lightning is paying or receiving money. In both cases, balance and history are not needed. And showing them right away is equivalent to a physical wallet that would be transparent, showing how much it contains.

Suggestion: when opening the app, there should just be three buttons:
Pay, Receive, and Balance.
Tapping the latter would be required before showing the balance.
Again, on the balance screen the history should not be shown by default, giving some information to a quick reader. History could also be a tap away. (or configurable)

**Comment by gotcha** [comment]
And after leaving the app and returning to it, UI should be reset to initial screen.

**Comment by kingonly** [comment]
We've added a optional PIN to address the privacy issues. Not sure we share the general approuch.

**Comment by gotcha** [comment]
The PIN does not, imo, avoid the other party to be able to take a look at the balance if it is displayed right after unlock...

Would you mind explaining your approach ? IOW, why showing balance right away is important ?

**Comment by kingonly** [comment]
Because it's a very common use case... Before you spend, you often want to understand how much you have. Most of the banking apps work in the same way. Even PayPal.
Mobile payment apps that work via credit card don't have access to your balance information. 
In a "be your own bank" approuch (or in Lightning "be your own checking account") we feel this information is useful.

**Comment by gotcha** [comment]
I really like your app so I absolutely do not mean to be offensive...
...but your answer sounds a bit like "because it has always been like this".

The proposal I made above enables checking balance before paying.

My banking app enables me to scan someone else QR without forcing me to access (and display) balance first.

**Comment by kingonly** [comment]
I'm open to discussion..
"it has always been like this" is important consideration... That's what users expect. To be honest, while I understand where you are coming from, I'm not sure it's a strong requirement many people share. IMHO, not strong enough to change the current design. I might be wrong and these things are very dynamic.

When designing the product, we've reviewed dozens of apps. We've also discussed with many potential users.
